cmd/removeEnumerator.go

```diff
@@ -73,6 +73,12 @@ func newRemoveEnumerator(cca *cookedCopyCmdArgs) (enumerator *copyEnumerator, er
 	finalize := func() error {
 		jobInitiated, err := transferScheduler.dispatchFinalPart()
 		if err != nil {
+			if strings.Contains(err.Error(), "scheduled") {
```

Comment:
This doesn't seem super-robust - i.e. when checking error message for string content, it will break if someone one day changes the error message text. Since the error will be coming from our own code, how about we define something like
```
var NothingScheduledError = errors.New("blah blah scheduled blah") // put the real text in here
```

Then the code that raises the error can return NothingScheduledError, and this code here can become

```
if err == NothingScheduledError
```

I believe that's the normal Golang pattern for this kind of thing. I'm not quite sure whether or how it should interact with the `ECopyJobPartOrderErrorType` stuff defined elsewhere - it seems like they are both talking about the same thing, so if that's the case, maybe this check here should check against the string `ECopyJobPartOrderErrorType.NoTransfersScheduledErr` instead of the NothingScheduledError I proposed above.